chef/cookbooks/cinder/recipes/volume.rb

```diff
@@ -231,12 +217,25 @@ def make_volume(node, backend_id, volume)
 end
 
 if have_rbd
-  include_recipe "ceph::cinder" if include_ceph_recipe
-
-  if node[:platform] == "suse"
-    package "ceph-common"
-    package "python-ceph"
+  if include_ceph_recipe
+    ceph_env_filter = " AND ceph_config_environment:ceph-config-default"
+    ceph_servers = search(:node, "roles:ceph-osd#{ceph_env_filter}") || []
+    if ceph_servers.length > 0
+      include_recipe "ceph::keyring"
+    else
+      message = "Ceph was not deployed with Crowbar yet!"
```

Comment:
We allow, please look on line 230. We are installing `ceph-common` and `python-ceph` client packages when we are going to use external Ceph cluster.